app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
+require 'sign_in/logger'
+
 module SignIn
   module Authentication
     extend ActiveSupport::Concern
 
     BEARER_PATTERN = /^Bearer /.freeze
+    VERSION_TAG = 'version:v0'
```

Comment:
I think for the rest of the sign in controller, version tag makes sense, but I'm not sure if it does for this specifically, since `authenticate` here will be used for any arbitrary authenticated route.

My recommendation would be to keep `VERSION_TAG` and related functionality in `sign_in_controller`, but to remove it here in `authenticate.rb`